src/app/cli/src/tests/full_test.ml

```diff
@@ -393,7 +393,7 @@ let run_test () : unit Deferred.t =
             (List.map accounts
                ~f:(fun ((keypair : Signature_lib.Keypair.t), account) ->
                  ( Public_key.compress keypair.public_key
-                 , account.Account.Poly.balance )))
+                 , account.Account.balance )))
         in
```

Comment:
it might be easier to read if the formal parameter were `(account : Account.t)`

qualifying the field name with the module name is a bit jarring